analysis/warriorarms/src/modules/features/RageTracker.js

```diff
@@ -24,7 +24,11 @@ class RageUsage extends ResourceTracker {
   }
 
   onDamage(event) {
-    this.processInvisibleEnergize(SPELLS.RAGE_AUTO_ATTACKS.id, RAGE_PER_MELEE_HIT);
+    if (event.hitType === 2) {
```

Comment:
you should use the `HIT_TYPE` enum (`import HIT_TYPES from 'game/HIT_TYPES'`) instead of a magic number here:

```suggestion
    if (event.hitType === HIT_TYPES.CRIT || event.hitType === HIT_TYPES.BLOCKED_CRIT) {
```

Reply:
Sure, I added HIT_TYPE.